apps/condo/domains/acquiring/schema/AcquiringIntegrationContext.js

```diff
@@ -2,26 +2,36 @@
  * Generated by `createschema acquiring.AcquiringIntegrationContext 'integration:Relationship:AcquiringIntegration:PROTECT; organization:Relationship:Organization:PROTECT; settings:Json; state:Json;' --force`
  */
 
-const { Text, Relationship } = require('@keystonejs/fields')
-const { isEmpty, isUndefined, isNull } = require('lodash')
+const { Text, Relationship, Select } = require('@keystonejs/fields')
```

Comment:
Remove all of it.

Use strings for KS5/KS6 compatibility

Reply:
Done